src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -1550,7 +1550,7 @@
     short-summary: The OS Type. Linux or Windows.
   - name: --os-sku
     type: string
-    short-summary: The OS SKU of the agent node pool. Ubuntu or CBLMariner for Linux. Windows2019 or Windows2022 for Windows.
+    short-summary: The OS SKU of the agent node pool. Ubuntu, CBLMariner or Ubuntu2204 for Linux. Windows2019 or Windows2022 for Windows.
```

Comment:
Is CBLMariner actually allowed here?

Maybe fix the docs?

Reply:
CBLMariner is no longer allowed: https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=/resourceprovider/server/microsoft.com/containerservice/server/validation/validators/ossku/linux_validator.go&version=GBmaster&_a=contents

Henry, one of the leads for Mariner has confirmed that we should drop CBLMariner and replace it with AzureLinux in the docs instead.

Reply:
updated the PR.